lib/rules/declaration-property-value-no-unknown/__tests__/index.mjs

```diff
@@ -11,6 +11,9 @@ testRule({
 		{
 			code: 'a { top: 0; }',
 		},
+		{
+			code: 'a { @starting-style { opacity: 0; } }',
+		},
```

Comment:
[suggest] This change also should fix the following *false negative* problem for `declaration-property-value-no-unknown`:

```css
a { @starting-style { padding: auto; } } /* no problem */

a { padding: auto; } /* problem for "auto" */
```

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYKACBnALkgTjgJYB2A5gLS4CeANjGigA5JRSlnwpICuOEAbhQBfEQB0SE1OhZsOXXvyHCQAGhAAzIvQBySALZxEMAB4Gm9AHRgsWNeAgktZBCGASUKMSHw96Wby53Ek9Pb1gwWgIkYkcKJnwIJhhCagoANyRaHhgKEggKHhIAa3yAd0kEFBxfGA9xEhV1SCciMgAxCHx9GNcAKyxHe1gmO0RgsJAaelpSHECvEABGADZLAAZLACZvCRVhIA)

So, how about opening another Pull Request for `declaration-property-value-no-unknown`?

Reply:
@ybiquitous thanks for clarifying it. I removed code related to `declaration-property-value-no-unknown`. Is there an opened issue for it?

Reply:
> So, how about opening another Pull Request for `declaration-property-value-no-unknown`?

I would prefer adding 2 changesets instead.

Reply:
> Is there an opened issue for it?

@fpetrakov No. It's OK to directly open a PR.

Reply:
#7461 was opened.